### PR TITLE
Table numbers off add finish state

### PR DESCRIPTION
**Pull request by iamsammak**
I rebased my original edits from old master to Andy's "add-finish-state" branch to account for the cleanup code changes.
Which is why this is set to merge to alau.add-finish-state

Added Chervon Up and Down buttons

screen shot 2019-02-05 at 8 31 24 pm

### CODE REVIEW

File: frontend/components/home.jsx [hunk 1]
```diff
@@ -4,26 +4,91 @@ import { Redirect } from 'react-router-dom';
...
+
+    this.addTens = this.addTens.bind(this);
+    this.addOnes = this.addOnes.bind(this);
+    this.setTable = this.setTable.bind(this);
+  }
+
+  addTens() {
+    let {tens} = this.state;
+
+    if (tens === 2) {
```
Comment (ihurrahi) [inline frontend/components/home.jsx]: this can be simplified to `(tens + 1) % 2`, same for below

File: frontend/components/home.jsx [hunk 2]
```diff
@@ -4,26 +4,91 @@ import { Redirect } from 'react-router-dom';
...
+      ones: 0
     };
+
+    this.addTens = this.addTens.bind(this);
+    this.addOnes = this.addOnes.bind(this);
+    this.setTable = this.setTable.bind(this);
+  }
+
+  addTens() {
+    let {tens} = this.state;
```
Comment (ihurrahi) [inline frontend/components/home.jsx]: use setState callback

**Comment by iamsammak** [comment]
Yeah I think I'll add the chervon arrows and add the functionality to go up or down

File: frontend/components/home.jsx [hunk 3]
```diff
@@ -2,28 +2,124 @@ import React from 'react';
...
+  padding: 0 2em;
+`;
+
+const DownIcon = styled(FontAwesomeIcon).attrs({
+  icon: 'chevron-down',
+})`
+  height: 25%;
+  font-size: 0.25em;
+  color: white;
+  padding: 0 2em;
```
Comment (ihurrahi) [inline frontend/components/home.jsx]: could use flexbox for the Containers?

File: frontend/components/home.jsx [hunk 4]
```diff
@@ -2,28 +2,124 @@ import React from 'react';
...
+    }
+
+    const { cookies } = this.props;
+    cookies.set("table", tableNumber);
+  }
+
   render() {
-    const { cookies, join } = this.props;
+    const {tens} = this.state;
+    const {ones} = this.state;
```
Comment (ihurrahi) [inline frontend/components/home.jsx]: can combine the two lines

**Pull request merged**